Prose/ProseLib/Sources/SidebarFeature/Footer/Footer.swift

```diff
@@ -86,28 +144,49 @@ struct FooterState: Equatable {
   var avatar: FooterAvatarState
   var actionButton: FooterActionMenuState
 
+  @BindableState var sheet: Sheet?
+
   init(
     credentials: UserCredentials,
     teamName: String = "Crisp",
     statusIcon: Character = "🚀",
     statusMessage: String = "Building new features.",
     avatar: FooterAvatarState = .init(avatar: .placeholder),
-    actionButton: FooterActionMenuState = .init()
+    actionButton: FooterActionMenuState = .init(),
+    sheet: Sheet? = nil
   ) {
     self.credentials = credentials
     self.teamName = teamName
     self.statusIcon = statusIcon
     self.statusMessage = statusMessage
     self.avatar = avatar
     self.actionButton = actionButton
+    self.sheet = sheet
+  }
+}
+
+public extension FooterState {
+  enum Sheet: Equatable {
+    case editProfile(EditProfileState)
```

Comment:
Are we expecting to show more sheets here or could this be a regular property?

Reply:
Yes, from "Account settings"

<img width="213" alt="Screenshot 2022-07-13 at 18 17 51" src="https://user-images.githubusercontent.com/37386490/178782225-c0068be5-365d-4b7c-a709-015bd7f0a5ac.png">